main/src/modules/reactiveLayer/behaviorManager/src/speech.cpp

```diff
@@ -0,0 +1,19 @@
+#include "speech.h"
+
+using namespace std;
+using namespace yarp::os;
+using namespace yarp::sig;
+
+void Speech::configure() {
+    ;
+}
+
+void Speech::run(Bottle args) {
```

Comment:
Hey lads, `Bottle args` is a completely different signature from `Bottle &args` and even from `const Bottle &args`. So, have your correct choice 😉